examples/web-wagmi/src/publications/UseCreatePost.tsx

```diff
@@ -1,7 +1,38 @@
+import { useFeed, useRecentPosts } from '@lens-protocol/react';
+
 import { LoginButton } from '../components/auth/LoginButton';
 import { WhenLoggedInWithProfile, WhenLoggedOut } from '../components/auth/auth';
-import { MyFeed } from './components/MyFeed';
+import { Loading } from '../components/loading/Loading';
+import { useInfiniteScroll } from '../hooks/useInfiniteScroll';
 import { PostComposer } from './components/PostComposer';
+import { PublicationCard } from './components/PublicationCard';
+
+type TimelineProps = {
+  profileId: string;
+};
+
+export function Timeline({ profileId }: TimelineProps) {
+  const recentPosts = useRecentPosts();
```

Comment:
do you think we could call the hook `useRecentPublications` to avoid having 2 hooks to maintain (or introducing breaking change in the next release by renaming this one)?

Reply:
I purposefully been very narrow to the task at hand cause I didn't want to worry about comments. We can introduce a breaking change in 0.x, no problems.

Maybe I would consider to have a `useRecentComment` hook even if under the hood it might/could leverage the same reactive var if need to.

The flows (create comment and create post) in our experience are very distinct and separate... I would carefully re-consider if worth to use the same reactive var or not.